src/bin/flamegraph.rs

```diff
@@ -174,6 +174,14 @@ struct Opt {
     )]
     search_color: SearchColor,
 
+    /// Stroke color
+    #[clap(
+        long = "stroke-color",
+        default_value = defaults::STROKE_COLOR,
+        value_name = "STRING"
```

Comment:
Why `STRING` and not `COLOR`?

Reply:
Same as https://github.com/jonhoo/inferno/blob/c9c07d2f0248e646438b4cba9eae15ef587791d5/src/bin/flamegraph.rs#L170-L175 and also because "none" is not a color but a valid entry for the stroke color.